roles/openshift_node/tasks/main.yml

```diff
@@ -94,6 +94,13 @@
     - openshift.common.use_openshift_sdn | default(true) | bool
     - not openshift.common.is_containerized | bool
 
+- name: Restart cri-o
+  systemd:
+    name: cri-o
+    enabled: yes
+    state: restarted
+  when: openshift_docker_use_crio | default(false)
```

Comment:
do you think we should move this at the end of the file and do only when `node_start_result | changed`?